include/SDL3/SDL_thread.h

```diff
@@ -263,7 +263,7 @@ extern SDL_DECLSPEC SDL_Thread * SDLCALL SDL_CreateThreadWithProperties(SDL_Prop
 
 /* The real implementation, hidden from the wiki, so it can show this as real functions that don't have macro magic. */
 #ifndef SDL_WIKI_DOCUMENTATION_SECTION
-#  if (defined(SDL_PLATFORM_WIN32) || defined(SDL_PLATFORM_GDK)) && !defined(SDL_PLATFORM_WINRT)
+#  if defined(SDL_PLATFORM_WINDOWS)
```

Comment:
Same here, PLATFORM_WINDOWS also includes PLATFORM_XBOXONE and PLATFORM_XBOXSERIES, which you don't want in this code. `defined(SDL_PLATFORM_WIN32) || defined(SDL_PLATFORM_GDK)` can't be broadly replaced with SDL_PLATFORM_WINDOWS.